### PR TITLE
Modelling comments

### PR DESCRIPTION
**Pull request by wamts**
I made some small changes to var-names in models and migrations as well.

### CODE REVIEW

File: server/models/entity_comment.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Entity_Comment = sequelize.define('Entity_Comment', {
+    comment: DataTypes.TEXT
+  }, {});
+  Entity_Comment.associate = function (models) {
+    Entity_Comment.belongsTo(models.Entity, { foreignKey: "entity_id" });
+    Entity_Comment.belongsTo(models.User, { foreignKey: "user_id" });
```
Comment (d-ang1993) [inline server/models/entity_comment.js:8]: Entity_Comment.belongsTo(models.User, { foreignKey: "user_id" }); indicates that there is an association between the User & Entity_Comment ? 

Meaning that this is a join table that holds both user_id & entity_id?

Comment (wamts) [reply server/models/entity_comment.js:8]: Entity_Comment is more specifically, a 'Comment' on an 'Entity'. Sorry for the confusing naming, Entity_Comment is not meant as a join table. I should probably rename it. I was just following the diagram here: https://docs.google.com/document/d/1Qrq4zKWv4ojhY300WW26UfULMh4TYPoNKzjQNMNqMNY/edit

A Comment is supposed to only have one User, hence line 8 (and lack of assoc in User model)
A Comment is only 'commented' under one Entity as well, hence line 7

Not exactly a join table, just assocs to both User and Entity

Comment (d-ang1993) [reply server/models/entity_comment.js:8]: ah my bad, I think this comment was suppose to be for the liked_entity.js

File: server/models/liked_entity.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Liked_Entity = sequelize.define('Liked_Entity', {
+    user_id: DataTypes.INTEGER,
+    entity_id: DataTypes.INTEGER
+  }, {});
+  Liked_Entity.associate = function(models) {
+    // associations can be defined here
```
Comment (d-ang1993) [inline server/models/liked_entity.js:8]: Same question here from above haha, is this a join table since it holds both user_id & entity_id

Comment (wamts) [reply server/models/liked_entity.js:8]: This table is supposed to be representing 'Likes'. It's a M2M relationship, so it's pretty much a join table.

File: server/models/entity_comment.js (hunk 1 above)
Comment (wamts) [reply server/models/entity_comment.js:8]: whoops my bad entity_comment would be a join table. I'll go ahead and fix the assocs.

Edit: Actually no it isn't exactly like user_skills though. Hmm...

Comment (d-ang1993) [reply server/models/entity_comment.js:8]: LOL xD still don't quite get this haha maybe we can ask Shums || Bonnie || Aceio?

**Pull request closed without merging**